composer.json

```diff
@@ -0,0 +1,34 @@
+{
+    "name": "pestphp/pest-dev-tools",
+    "description": "Dev requirements for Pest, The elegant PHP Testing Framework.",
+    "keywords": [
+        "php",
+        "framework",
+        "pest",
+        "unit",
+        "test",
+        "testing"
+    ],
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Nuno Maduro",
+            "email": "enunomaduro@gmail.com"
+        },
+        {
+            "name": "Didier Sampaolo",
+            "email": "didier@lvlup.fr"
+        }
+    ],
+    "require-dev": {
```

Comment:
I think it should be in `require` section. 

When we require a package, it never comes with dev dependencies, even when we require a package as dev dependencies.

You can get inspired by [metapackage](https://packagist.org/?type=metapackage) or [symfony-pack](https://packagist.org/?type=symfony-pack) on packagist :wink:

Reply:
Yep, I think once this is changed then this PR should be ready to merge. 🎉